### PR TITLE
Add Microsoft Edge browser

### PR DESCRIPTION
**Pull request by KyleKing**
Added a mode `'edge'` that supports Microsoft Edge. The new mode handles creating the CLI command `start microsoft-edge:<URL>` to launch a new Edge window or tab. For a corporate Windows 10 environment, this is a really lightweight solution to support users who don't install Chrome.

Right now, Edge ships with all Windows 10 (but may become optional sometime in the future) and it can be run in a Win10 VM on prior Windows releases. To show how to use this mode, I've added an alternate `example 01` and updated `example 07` to fall back to Edge on Win10 if Chrome isn't installed.

FYI: [Microsoft apparently has plans to expand Edge to Windows 7, WIndows 8, & MacOS](https://blogs.windows.com/windowsexperience/2018/12/06/microsoft-edge-making-the-web-better-through-more-open-source-collaboration/)

### CODE REVIEW

**Comment by ChrisKnott** [comment]
Kyle, this is a great idea and a good point about corporate environments without Chrome. Let me just clear the PR backlog and I will come back to this.

Also, thanks for your interest in the project and contributions on other Issues.

**Comment by KyleKing** [comment]
@ChrisKnott, I can update the fork to resolve conflicts. Just let me know when the main repository is stable enough and you would be ready to merge

**Comment by ChrisKnott** [comment]
@KyleKing Finally pushed changes to API, as it was taking to long for me to add new features. If you look at the new code for `eel.browsers`, `eel.chrome` and `eel.electron` you should see how to make it fit with the new codebase.

Sorry for slow response, thanks again for your help with the project!

**Comment by KyleKing** [comment]
Tested on Windows 10. I think this is ready to go!

File: README.md [hunk 1]
```diff
@@ -316,3 +317,10 @@ If you want to package your app into a program that can be run on a computer wit
 6. When happy that your app is working correctly, add `--onefile --noconsole` flags to build a single executable file
 
 Consult the [documentation for PyInstaller](http://PyInstaller.readthedocs.io/en/stable/) for more options.
+
+## Microsoft Edge
+
+For Windows 10 users, Microsoft Edge (`eel.start(.., mode='edge')`) is installed by default and a useful fallback if a preffered browser is not installed. See the examples:
```
Comment (samuelhwilliams) [inline README.md]:
```suggestion
For Windows 10 users, Microsoft Edge (`eel.start(.., mode='edge')`) is installed by default and a useful fallback if a preferred browser is not installed. See the examples:
```

File: README.md [hunk 2]
```diff
@@ -316,3 +317,10 @@ If you want to package your app into a program that can be run on a computer wit
...
+
+- A Hello World example using Micorosft Edge: [examples/01 - hello_world-Edge/](https://github.com/ChrisKnott/Eel/tree/master/examples/01%20-%20hello_world-Edge)
```
Comment (samuelhwilliams) [inline README.md]:
```suggestion
- A Hello World example using Microsoft Edge: [examples/01 - hello_world-Edge/](https://github.com/ChrisKnott/Eel/tree/master/examples/01%20-%20hello_world-Edge)
```

**Pull request merged**